### PR TITLE
Updating the Russian translation

### PR DESCRIPTION
**Pull request by Mauladen**
The translation update

### CODE REVIEW

File: locale/ru.po [hunk 1]
```diff
@@ -108,7 +108,7 @@ msgstr "Получение всех данных из AUR"
 
 #: pikaur/install_cli.py:167
 msgid "Ignoring package {}"
-msgstr "Игнорирую пакет {}"
+msgstr "Игнорирование пакета {}"
```
Comment (Yardanico) [inline locale/ru.po]: @actionless disagreed with this one already

File: locale/ru.po [hunk 2]
```diff
@@ -153,7 +153,7 @@ msgstr "Продолжить установку? [Y/n]"
 
 #: pikaur/package_update.py:45
 msgid "Reading AUR packages info..."
-msgstr "Чтение информации пакетов из AUR..."
+msgstr "Чтение информации о пакетах из AUR..."
```
Comment (Yardanico) [inline locale/ru.po]: I think we should have support for plurals here (in src) :)

File: locale/ru.po [hunk 3]
```diff
@@ -291,15 +291,11 @@ msgstr "m"
...
-"редактор не найден. Попробуйте установить переменные окружения $VISUAL или "
-"$EDITOR."
+msgstr "редактор не найден. Попробуйте установить переменную окружения $VISUAL или $EDITOR."
 
 #: pikaur/main.py:244
 msgid "pikaur requires systemd >= 235 (dynamic users) to be run as root."
-msgstr ""
-"pikaur требует systemd >= 235 (поддержка динамических пользователей) для "
-"запуска от root."
+msgstr "pikaur требует systemd >= 235 (поддержка динамических пользователей) для запуска от root."
```
Comment (Yardanico) [inline locale/ru.po]: Well, I wanted to do these lines like this too, but I wanted to have less than 80 chars on one line

**Comment by Yardanico** [comment]
And yeah, @Mauladen, you said that this is not a good translation, but you mostly have word reorderings/changing number of lines, so your statement wasn't true :)

File: locale/ru.po (hunk 1 above)
Comment (Mauladen) [reply locale/ru.po]: @Yardanico I was translating for Manjaro Architect and it seems to me that I have the experience to claim my rightness

File: locale/ru.po (hunk 2 above)
Comment (Mauladen) [reply locale/ru.po]: @Yardanico And I did it removed plural?

File: locale/ru.po (hunk 1 above)
Comment (Yardanico) [reply locale/ru.po]: @Mauladen lol, why would it depend? This is another project, not `Manjaro Architect`

File: locale/ru.po (hunk 2 above)
Comment (Yardanico) [reply locale/ru.po]: @Mauladen I've meant that we maybe need add support for one/many packages in the actual pikaur source code for this line

File: locale/ru.po (hunk 1 above)
Comment (Yardanico) [reply locale/ru.po]: @Mauladen well, I looked at your Manjaro Architect translations - there's a lot of bad translations - https://github.com/Chrysostomus/manjaro-architect/commit/e8a25e08d671320f3a75fd7e6669f1d145a6ee19#diff-dc81ed930c57db3a8c05b0af3d6764c6
Look at `SysRescBody` here for example

Comment (Yardanico) [reply locale/ru.po]: And translating `Manjaro` to `Манждаро` is over the top.. You surely have experience to claim your rightness

Comment (Mauladen) [reply locale/ru.po]: I agree, the translation of this particular line is not very good
But the only thing I did in your translation was to change the structure

File: locale/ru.po (hunk 3 above)
Comment (Mauladen) [reply locale/ru.po]: @Yardanico I think the terminal will not have problems with the transfer of text

File: locale/ru.po (hunk 1 above)
Comment (Yardanico) [reply locale/ru.po]: @Mauladen why did you call it a `bad translation` then? :)

Comment (actionless) [reply locale/ru.po]: `пакет {} (был?) проигнорирован`

`игнорирование` in russian can be only a noun:
https://en.wiktionary.org/wiki/%D0%B8%D0%B3%D0%BD%D0%BE%D1%80%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5

File: locale/ru.po (hunk 2 above)
Comment (actionless) [reply locale/ru.po]: 551c9f4ddf0a95e9a68709ed42ac122dfe3cf586

File: locale/ru.po (hunk 3 above)
Comment (actionless) [reply locale/ru.po]: doesn't really matter, i have quite strict requirements only to code

File: locale/ru.po (hunk 1 above)
Comment (Mauladen) [reply locale/ru.po]: @actionless Here you are right

File: locale/ru.po (hunk 2 above)
Comment (Mauladen) [reply locale/ru.po]: @actionless I ask you to accept these changes, and will finalize the rest of @Yardanico

**Comment by actionless** [comment]
thanks / спасибо!

**Pull request merged**